subscriptions/subscriptions.go

```diff
@@ -110,11 +110,6 @@ func (p *Subscriptions) HasTxPermission(
 		return false, fmt.Errorf("organization is nil")
 	}
 
-	// Check if the organization has a subscription
-	if org.Subscription.PlanID == 0 {
-		return false, fmt.Errorf("organization has no subscription plan")
-	}
-
```

Comment:
TBD, i removed this since it looks like a leftover,
but please review carefully